src/System Application/Test/Azure Key Vault/src/AzureKeyVaultTest.Codeunit.al

```diff
@@ -95,11 +96,12 @@ codeunit 135212 "Azure Key Vault Test"
 
         // [GIVEN] A configured Azure Key Vault
         MockAzureKeyvaultSecretProvider := MockAzureKeyvaultSecretProvider.MockAzureKeyVaultSecretProvider();
+        MockAzureKeyvaultSecretProvider.AddSecretMapping('some-secret', 'SecretFromKeyVault');
         AzureKeyVaultTestLibrary.SetAzureKeyVaultSecretProvider(MockAzureKeyvaultSecretProvider);
-        AzureKeyVaultTestLibrary.ClearSecrets();
 
         // [WHEN] The key vault is called with an unknown key
         asserterror AzureKeyVault.GetAzureKeyVaultSecret('somekeythatdoesnotexist', Secret);
+        Assert.ExpectedError(StrSubstNo(MissingSecretErr, 'somekeythatdoesnotexist'));
```

Comment:
You should also test a secret that does exist but is blank. After all that's the functionality you are adding above. This always failed after all. I'm actually surprised you now get the new error since the failure should be on GetSecretFromClient (just like it failed before..)

Reply:
Funny enough, this is literally a blank scenario, as I said above platform would return an empty string for a non existing secret.